examples/CRISP/scripts/dev_cipher.sh

```
@@ -6,9 +6,15 @@ set -euo pipefail
 rm -rf ./enclave/data
 rm -rf ./enclave/config
 
-PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
-
-enclave wallet set --name ag --private-key "$PRIVATE_KEY" 
+PRIVATE_KEY_AG="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
```

Shellcheck (0.11.0)

[warning] PRIVATE_KEY_AG appears unused. Verify use (or export if used externally).

(SC2034)